app/models/pame_evaluation.rb

```diff
@@ -89,14 +89,14 @@ def self.parse_filters(filters)
       when 'iso3'
         countries = options
         site_ids << countries.map{ |iso3| Country.find_by(iso_3: iso3).protected_areas.pluck(:id) }
-        where_params[:sites] = options.empty? ? nil : "protected_area_id IN (#{site_ids.join(',')})"
+        where_params[:sites] = site_ids.empty? ? nil : "protected_area_id IN (#{site_ids.join(',')})"
         country_ids << countries.map{ |iso3| "(#{ Country.find_by(iso_3: iso3).id })" }
```

Comment:
`country_ids` has ids surrounded by parenthesis (e.g. (294)), so that would probably return an error within the SQL statement.
Is there a reason for those parentheses? Does this functionality work for you at all?

In the meantime, I'm also trying to understand why `country.protected_areas` returns an empty collection for me, while I think it should return something

Reply:
Oh I see @Levia . For some reason I needed those for the query to work. I will try it again.

Reply:
Do you have a production database dump in your local postgres for PP?

Reply:
I managed to fix that last bit now. It was loading a wrong database.

Reply:
Oh great. I am glad that part is fixed now. I just managed to break it completely:
```
ActiveRecord::StatementInvalid (PG::SyntaxError: ERROR:  syntax error at or near "8"
LINE 1: ..._evaluations"."country_id" WHERE (countries.id IN 8,2))) AS ...
```
It was working before for me. So I will need to resolve this.

Reply:
If you keep your original code and test it by selecting multiple countries in the filter, you should see it broken.
The issue is that you have surrounded every id with parantheses in the `map` statement, while you should do it when assigning the value to the `where_params[:iso3]` variable, like you did for the `sites_ids`

Reply:
Thanks I think I have fixed that!

Reply:
Looks good now!